core/layer_manager.ts

```diff
@@ -48,6 +51,35 @@ export class LayerManager {
     return dom.createSvgElement(Svg.G, {}, svg);
   }
 
+  private createAnimationLayer(injectionDiv: Element) {
```

Comment:
Please provide a suitable JSDoc for this (even if it's just an excerpt of the one from `appendToAnimationLayer`).

Reply:
This is private so it doesn't need JSDoc. I think the method signature makes it pretty clear what it's doing.